src/BusinessService.ts

```diff
@@ -229,6 +229,11 @@ export type BusinessUser = {
    * `undefined` if the user is not joined via invitation.
    */
   invitationCreatedAt?: string;
+
+  /**
+   * The users ID in business
+   */
+  businessUserId?: string;
```

Comment:
Do we have this already on backend's main branch? I can't find this key being returned from `GET /business/:id/users`

Reply:
Yes. Adding it here https://github.com/lensapp/lenscloud/pull/2462

Reply:
Ah, so it’s not on main yet